tests/integration/test_web_client.py

```diff
@@ -183,6 +186,31 @@ def test_creation_lifecycle(self, antares_web: AntaresWebDesktop):
         assert storage_fr.name == st_storage_name
         assert storage_fr.id == "cluster_test"
 
+        # testing
```

Comment:
Change this message to a more explicit one explaining which tests you're performing.